spec/workers/push_entity_worker_spec.rb

```diff
@@ -269,6 +287,24 @@
               worker.push_entity_to_global_registry
               requests.each { |r| expect(r).to have_been_requested.once }
               expect(person.global_registry_id).to eq '22527d88-3cba-11e7-b876-129bd0521531'
+              expect(person.global_registry_fingerprint).to eq '4c671c203b5dd19cdc1920ba5434cf64'
+            end
+          end
+
+          context 'fingerprints match' do
+            it 'should do nothing' do
+              request = stub_request(:put,
+                                     'https://backend.global-registry.org/entities/f8d20318-2ff2-4a98-a5eb-e9d840508bf1')
```

Comment:
This url and body expectation could be DRYed up to some common variables if you want to :)